mlflow/sagemaker/cli.py

```diff
@@ -588,6 +588,12 @@ def setup_container(_):
             return "\n".join(
                 [
                     'ENV {disable_env}="false"',
+                    # Explicitly install protobuf==3.20.1 to replace any potential protobuf >= 4.0
+                    # versions preinstalled in the model serving container. This is necessary for
+                    # compatibility with MLflow versions <= 1.26.0, which are incompatible with
+                    # protobuf >= 4.0 but do not explicitly require protobuf < 4.0 in the package
+                    # requirements
+                    "RUN pip install protobuf==3.20.1",
```

Comment:
Necessary in order for the subsequent `import mlflow` to succeed.

Reply:
Question for my own understanding: Why do we need both:
(1) `_install_pyfunc_deps` installs `protobuf==3.20.1` (above)
and
(2) `setup_container` installs `protobuf==3.20.1` (here)

I would think that (2) would be sufficient

Reply:
Got it! Thanks for explaining this.